python/py-xraylarch/Portfile

```diff
@@ -28,14 +28,14 @@ long_description    Larch is a open-source library and set of applications for p
 homepage            https://xraypy.github.io/xraylarch
 
 # Find the latest at https://pypi.org/project/xraylarch/#files
-master_sites        https://files.pythonhosted.org/packages/ff/4a/cf7f494aaa3ea1ba15adf5806cef0ac5136affc7e8f7c2048999fa4e28d9/
+master_sites        https://files.pythonhosted.org/packages/69/ee/25315817d35756a170df5c022f3d2ec550a599df927a6b1e02b3079dbcef/
```

Comment:
why again are we not building this package in the normal way but instead download a wheel? That is really not how it should be done...

Reply:
Good question. In PR #15083, we found that the upstream source did not distribute tarballs. I can't remember why, but it does distribute wheel files, so I proposed that we might go that way.

Reply:
it does provide a tarball on PyPi now so please switch to that.

Reply:
Good catch. Changing the PR to use the tarball now.